lib/src/widgets/yaru_switch_theme.dart

```diff
@@ -0,0 +1,125 @@
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+
+@immutable
+class YaruSwitchThemeData extends ThemeExtension<YaruSwitchThemeData>
+    with Diagnosticable {
+  const YaruSwitchThemeData({
+    this.color,
+    this.borderColor,
+    this.dotColor,
+    this.indicatorColor,
+  });
+
+  final MaterialStateProperty<Color?>? color;
+  final MaterialStateProperty<Color?>? borderColor;
+  final MaterialStateProperty<Color?>? dotColor;
+  final MaterialStateProperty<Color?>? indicatorColor;
```

Comment:
could these be called "thumb" and "track" like in flutter's SwitchThemeData?

Reply:
I agree for the thumb :+1: